core-primitives/enclave-api/ffi/src/lib.rs

```diff
@@ -4,7 +4,14 @@ use sgx_types::{c_int, sgx_enclave_id_t, sgx_quote_sign_type_t, sgx_status_t};
 
 extern "C" {
 
-	pub fn init(eid: sgx_enclave_id_t, retval: *mut sgx_status_t) -> sgx_status_t;
+	pub fn init(
```

Comment:
The `init` function now initializes an enclave primitives storage, storing (for now, might be expanded?) worker / enclave urls not known by the pallet.